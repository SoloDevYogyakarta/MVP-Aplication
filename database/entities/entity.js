"use strict";
var __importDefault = (this && this.__importDefault) || function (mod) {
    return (mod && mod.__esModule) ? mod : { "default": mod };
};
Object.defineProperty(exports, "__esModule", { value: true });
exports.sequelize = void 0;
const dotenv_1 = __importDefault(require("dotenv"));
const sequelize_1 = require("sequelize");
dotenv_1.default.config();
const env = process.env;
exports.sequelize = new sequelize_1.Sequelize(env['DB_NAME'], env['DB_USER'], env['DB_PASS'], {
    host: env['DB_HOST'],
    port: Number(env['DB_PORT']),
    logging: false,
    database: env['DB_NAME'],
    dialect: env['DB_DIALECT'],
});
//# sourceMappingURL=entity.js.map