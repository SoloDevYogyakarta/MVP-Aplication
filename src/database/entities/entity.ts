import dotenv from 'dotenv';
import { Dialect, Sequelize } from 'sequelize';

dotenv.config();

const env = process.env;

export const sequelize = new Sequelize(
  env['DB_NAME'],
  env['DB_USER'],
  env['DB_PASS'],
  {
    host: env['DB_HOST'],
    port: Number(env['DB_PORT']),
    logging: false,
    database: env['DB_NAME'],
    dialect: env['DB_DIALECT'] as Dialect,
  },
);
