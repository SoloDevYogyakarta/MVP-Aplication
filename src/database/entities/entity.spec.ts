import { sequelize } from './entity';

describe('Entity', () => {
  it('should to be defined', () => expect(sequelize).toBeDefined());

  it('render correctly', () => expect(sequelize).toMatchSnapshot());

  it('connect to database', async () => {
    await sequelize
      .authenticate()
      .then(() => expect(true).toEqual(true))
      .catch((err) => {
        throw new Error(err);
      });
  });
});
