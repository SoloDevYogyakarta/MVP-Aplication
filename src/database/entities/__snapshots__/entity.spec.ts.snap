// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Entity render correctly 1`] = `
Sequelize {
  "config": {
    "database": "mvpapplication",
    "dialectModule": null,
    "dialectModulePath": null,
    "dialectOptions": undefined,
    "host": "127.0.0.1",
    "keepDefaultTimezone": undefined,
    "native": false,
    "password": "password",
    "pool": {},
    "port": 5432,
    "protocol": "tcp",
    "replication": false,
    "ssl": undefined,
    "username": "root",
  },
  "connectionManager": ConnectionManager {
    "config": {
      "database": "mvpapplication",
      "dialectModule": null,
      "dialectModulePath": null,
      "dialectOptions": undefined,
      "host": "127.0.0.1",
      "keepDefaultTimezone": undefined,
      "native": false,
      "password": "password",
      "pool": {
        "acquire": 60000,
        "evict": 1000,
        "idle": 10000,
        "max": 5,
        "min": 0,
        "validate": [Function],
      },
      "port": 5432,
      "protocol": "tcp",
      "replication": false,
      "ssl": undefined,
      "username": "root",
    },
    "dialect": PostgresDialect {
      "connectionManager": [Circular],
      "queryGenerator": PostgresQueryGenerator {
        "_dialect": [Circular],
        "_quoteIdentifier": [Function],
        "dialect": "postgres",
        "options": {
          "attributeBehavior": "throw",
          "benchmark": false,
          "database": "mvpapplication",
          "databaseVersion": 0,
          "define": {},
          "dialect": "postgres",
          "dialectModule": null,
          "dialectModulePath": null,
          "hooks": {},
          "host": "127.0.0.1",
          "isolationLevel": null,
          "logQueryParameters": false,
          "logging": false,
          "minifyAliases": false,
          "native": false,
          "omitNull": false,
          "pool": {},
          "port": 5432,
          "protocol": "tcp",
          "query": {},
          "quoteIdentifiers": true,
          "replication": false,
          "retry": {
            "match": [
              "SQLITE_BUSY: database is locked",
            ],
            "max": 5,
          },
          "ssl": undefined,
          "standardConformingStrings": true,
          "storage": "/Users/kenedy-/projects/mvpapplication/src/utils/mvpapplication.sqlite",
          "sync": {},
          "timezone": "+00:00",
          "transactionType": "DEFERRED",
          "typeValidation": false,
        },
        "quoteIdentifier": [Function],
        "sequelize": [Circular],
        "typeValidation": undefined,
      },
      "queryInterface": PostgresQueryInterface {
        "queryGenerator": PostgresQueryGenerator {
          "_dialect": [Circular],
          "_quoteIdentifier": [Function],
          "dialect": "postgres",
          "options": {
            "attributeBehavior": "throw",
            "benchmark": false,
            "database": "mvpapplication",
            "databaseVersion": 0,
            "define": {},
            "dialect": "postgres",
            "dialectModule": null,
            "dialectModulePath": null,
            "hooks": {},
            "host": "127.0.0.1",
            "isolationLevel": null,
            "logQueryParameters": false,
            "logging": false,
            "minifyAliases": false,
            "native": false,
            "omitNull": false,
            "pool": {},
            "port": 5432,
            "protocol": "tcp",
            "query": {},
            "quoteIdentifiers": true,
            "replication": false,
            "retry": {
              "match": [
                "SQLITE_BUSY: database is locked",
              ],
              "max": 5,
            },
            "ssl": undefined,
            "standardConformingStrings": true,
            "storage": "/Users/kenedy-/projects/mvpapplication/src/utils/mvpapplication.sqlite",
            "sync": {},
            "timezone": "+00:00",
            "transactionType": "DEFERRED",
            "typeValidation": false,
          },
          "quoteIdentifier": [Function],
          "sequelize": [Circular],
          "typeValidation": undefined,
        },
        "sequelize": [Circular],
      },
      "sequelize": [Circular],
    },
    "dialectName": "postgres",
    "enumOids": {
      "arrayOids": [],
      "oids": [],
    },
    "lib": PG {
      "Client": [Function],
      "Connection": [Function],
      "DatabaseError": [Function],
      "Pool": [Function],
      "Query": [Function],
      "_pools": [],
      "defaults": {
        "application_name": undefined,
        "binary": false,
        "client_encoding": "",
        "connect_timeout": 0,
        "connectionString": undefined,
        "database": undefined,
        "fallback_application_name": undefined,
        "host": "localhost",
        "idleTimeoutMillis": 30000,
        "idle_in_transaction_session_timeout": false,
        "keepalives": 1,
        "keepalives_idle": 0,
        "lock_timeout": false,
        "max": 10,
        "options": undefined,
        "parseInputDatesAsUTC": false,
        "parseInt8": undefined,
        "password": null,
        "port": 5432,
        "query_timeout": false,
        "rows": 0,
        "ssl": false,
        "statement_timeout": false,
        "user": "kenedy-",
      },
      "escapeIdentifier": [Function],
      "escapeLiteral": [Function],
      "types": {
        "arrayParser": {
          "create": [Function],
        },
        "builtins": {
          "ABSTIME": 702,
          "ACLITEM": 1033,
          "BIT": 1560,
          "BOOL": 16,
          "BPCHAR": 1042,
          "BYTEA": 17,
          "CHAR": 18,
          "CID": 29,
          "CIDR": 650,
          "CIRCLE": 718,
          "DATE": 1082,
          "FLOAT4": 700,
          "FLOAT8": 701,
          "GTSVECTOR": 3642,
          "INET": 869,
          "INT2": 21,
          "INT4": 23,
          "INT8": 20,
          "INTERVAL": 1186,
          "JSON": 114,
          "JSONB": 3802,
          "MACADDR": 829,
          "MACADDR8": 774,
          "MONEY": 790,
          "NUMERIC": 1700,
          "OID": 26,
          "PATH": 602,
          "PG_DEPENDENCIES": 3402,
          "PG_LSN": 3220,
          "PG_NDISTINCT": 3361,
          "PG_NODE_TREE": 194,
          "POLYGON": 604,
          "REFCURSOR": 1790,
          "REGCLASS": 2205,
          "REGCONFIG": 3734,
          "REGDICTIONARY": 3769,
          "REGNAMESPACE": 4089,
          "REGOPER": 2203,
          "REGOPERATOR": 2204,
          "REGPROC": 24,
          "REGPROCEDURE": 2202,
          "REGROLE": 4096,
          "REGTYPE": 2206,
          "RELTIME": 703,
          "SMGR": 210,
          "TEXT": 25,
          "TID": 27,
          "TIME": 1083,
          "TIMESTAMP": 1114,
          "TIMESTAMPTZ": 1184,
          "TIMETZ": 1266,
          "TINTERVAL": 704,
          "TSQUERY": 3615,
          "TSVECTOR": 3614,
          "TXID_SNAPSHOT": 2970,
          "UUID": 2950,
          "VARBIT": 1562,
          "VARCHAR": 1043,
          "XID": 28,
          "XML": 142,
        },
        "getTypeParser": [Function],
        "setTypeParser": [Function],
      },
    },
    "nameOidMap": {},
    "oidParserMap": Map {},
    "pool": Pool {
      "_availableObjects": [],
      "_count": 0,
      "_draining": false,
      "_factory": {
        "acquireTimeoutMillis": 60000,
        "create": [Function],
        "destroy": [Function],
        "idleTimeoutMillis": 10000,
        "max": 5,
        "maxUses": undefined,
        "min": 0,
        "name": "sequelize",
        "reapIntervalMillis": 1000,
        "validate": [Function],
      },
      "_inUseObjects": [],
      "_pendingAcquires": [],
      "_removeIdleScheduled": false,
      "acquireTimeoutMillis": 60000,
      "idleTimeoutMillis": 10000,
      "log": false,
      "maxUsesPerResource": Infinity,
      "reapIntervalMillis": 1000,
    },
    "sequelize": [Circular],
    "versionPromise": null,
  },
  "dialect": PostgresDialect {
    "connectionManager": ConnectionManager {
      "config": {
        "database": "mvpapplication",
        "dialectModule": null,
        "dialectModulePath": null,
        "dialectOptions": undefined,
        "host": "127.0.0.1",
        "keepDefaultTimezone": undefined,
        "native": false,
        "password": "password",
        "pool": {
          "acquire": 60000,
          "evict": 1000,
          "idle": 10000,
          "max": 5,
          "min": 0,
          "validate": [Function],
        },
        "port": 5432,
        "protocol": "tcp",
        "replication": false,
        "ssl": undefined,
        "username": "root",
      },
      "dialect": [Circular],
      "dialectName": "postgres",
      "enumOids": {
        "arrayOids": [],
        "oids": [],
      },
      "lib": PG {
        "Client": [Function],
        "Connection": [Function],
        "DatabaseError": [Function],
        "Pool": [Function],
        "Query": [Function],
        "_pools": [],
        "defaults": {
          "application_name": undefined,
          "binary": false,
          "client_encoding": "",
          "connect_timeout": 0,
          "connectionString": undefined,
          "database": undefined,
          "fallback_application_name": undefined,
          "host": "localhost",
          "idleTimeoutMillis": 30000,
          "idle_in_transaction_session_timeout": false,
          "keepalives": 1,
          "keepalives_idle": 0,
          "lock_timeout": false,
          "max": 10,
          "options": undefined,
          "parseInputDatesAsUTC": false,
          "parseInt8": undefined,
          "password": null,
          "port": 5432,
          "query_timeout": false,
          "rows": 0,
          "ssl": false,
          "statement_timeout": false,
          "user": "kenedy-",
        },
        "escapeIdentifier": [Function],
        "escapeLiteral": [Function],
        "types": {
          "arrayParser": {
            "create": [Function],
          },
          "builtins": {
            "ABSTIME": 702,
            "ACLITEM": 1033,
            "BIT": 1560,
            "BOOL": 16,
            "BPCHAR": 1042,
            "BYTEA": 17,
            "CHAR": 18,
            "CID": 29,
            "CIDR": 650,
            "CIRCLE": 718,
            "DATE": 1082,
            "FLOAT4": 700,
            "FLOAT8": 701,
            "GTSVECTOR": 3642,
            "INET": 869,
            "INT2": 21,
            "INT4": 23,
            "INT8": 20,
            "INTERVAL": 1186,
            "JSON": 114,
            "JSONB": 3802,
            "MACADDR": 829,
            "MACADDR8": 774,
            "MONEY": 790,
            "NUMERIC": 1700,
            "OID": 26,
            "PATH": 602,
            "PG_DEPENDENCIES": 3402,
            "PG_LSN": 3220,
            "PG_NDISTINCT": 3361,
            "PG_NODE_TREE": 194,
            "POLYGON": 604,
            "REFCURSOR": 1790,
            "REGCLASS": 2205,
            "REGCONFIG": 3734,
            "REGDICTIONARY": 3769,
            "REGNAMESPACE": 4089,
            "REGOPER": 2203,
            "REGOPERATOR": 2204,
            "REGPROC": 24,
            "REGPROCEDURE": 2202,
            "REGROLE": 4096,
            "REGTYPE": 2206,
            "RELTIME": 703,
            "SMGR": 210,
            "TEXT": 25,
            "TID": 27,
            "TIME": 1083,
            "TIMESTAMP": 1114,
            "TIMESTAMPTZ": 1184,
            "TIMETZ": 1266,
            "TINTERVAL": 704,
            "TSQUERY": 3615,
            "TSVECTOR": 3614,
            "TXID_SNAPSHOT": 2970,
            "UUID": 2950,
            "VARBIT": 1562,
            "VARCHAR": 1043,
            "XID": 28,
            "XML": 142,
          },
          "getTypeParser": [Function],
          "setTypeParser": [Function],
        },
      },
      "nameOidMap": {},
      "oidParserMap": Map {},
      "pool": Pool {
        "_availableObjects": [],
        "_count": 0,
        "_draining": false,
        "_factory": {
          "acquireTimeoutMillis": 60000,
          "create": [Function],
          "destroy": [Function],
          "idleTimeoutMillis": 10000,
          "max": 5,
          "maxUses": undefined,
          "min": 0,
          "name": "sequelize",
          "reapIntervalMillis": 1000,
          "validate": [Function],
        },
        "_inUseObjects": [],
        "_pendingAcquires": [],
        "_removeIdleScheduled": false,
        "acquireTimeoutMillis": 60000,
        "idleTimeoutMillis": 10000,
        "log": false,
        "maxUsesPerResource": Infinity,
        "reapIntervalMillis": 1000,
      },
      "sequelize": [Circular],
      "versionPromise": null,
    },
    "queryGenerator": PostgresQueryGenerator {
      "_dialect": [Circular],
      "_quoteIdentifier": [Function],
      "dialect": "postgres",
      "options": {
        "attributeBehavior": "throw",
        "benchmark": false,
        "database": "mvpapplication",
        "databaseVersion": 0,
        "define": {},
        "dialect": "postgres",
        "dialectModule": null,
        "dialectModulePath": null,
        "hooks": {},
        "host": "127.0.0.1",
        "isolationLevel": null,
        "logQueryParameters": false,
        "logging": false,
        "minifyAliases": false,
        "native": false,
        "omitNull": false,
        "pool": {},
        "port": 5432,
        "protocol": "tcp",
        "query": {},
        "quoteIdentifiers": true,
        "replication": false,
        "retry": {
          "match": [
            "SQLITE_BUSY: database is locked",
          ],
          "max": 5,
        },
        "ssl": undefined,
        "standardConformingStrings": true,
        "storage": "/Users/kenedy-/projects/mvpapplication/src/utils/mvpapplication.sqlite",
        "sync": {},
        "timezone": "+00:00",
        "transactionType": "DEFERRED",
        "typeValidation": false,
      },
      "quoteIdentifier": [Function],
      "sequelize": [Circular],
      "typeValidation": undefined,
    },
    "queryInterface": PostgresQueryInterface {
      "queryGenerator": PostgresQueryGenerator {
        "_dialect": [Circular],
        "_quoteIdentifier": [Function],
        "dialect": "postgres",
        "options": {
          "attributeBehavior": "throw",
          "benchmark": false,
          "database": "mvpapplication",
          "databaseVersion": 0,
          "define": {},
          "dialect": "postgres",
          "dialectModule": null,
          "dialectModulePath": null,
          "hooks": {},
          "host": "127.0.0.1",
          "isolationLevel": null,
          "logQueryParameters": false,
          "logging": false,
          "minifyAliases": false,
          "native": false,
          "omitNull": false,
          "pool": {},
          "port": 5432,
          "protocol": "tcp",
          "query": {},
          "quoteIdentifiers": true,
          "replication": false,
          "retry": {
            "match": [
              "SQLITE_BUSY: database is locked",
            ],
            "max": 5,
          },
          "ssl": undefined,
          "standardConformingStrings": true,
          "storage": "/Users/kenedy-/projects/mvpapplication/src/utils/mvpapplication.sqlite",
          "sync": {},
          "timezone": "+00:00",
          "transactionType": "DEFERRED",
          "typeValidation": false,
        },
        "quoteIdentifier": [Function],
        "sequelize": [Circular],
        "typeValidation": undefined,
      },
      "sequelize": [Circular],
    },
    "sequelize": [Circular],
  },
  "modelManager": ModelManager {
    "models": [],
    "sequelize": [Circular],
  },
  "models": {},
  "options": {
    "attributeBehavior": "throw",
    "benchmark": false,
    "database": "mvpapplication",
    "databaseVersion": 0,
    "define": {},
    "dialect": "postgres",
    "dialectModule": null,
    "dialectModulePath": null,
    "hooks": {},
    "host": "127.0.0.1",
    "isolationLevel": null,
    "logQueryParameters": false,
    "logging": false,
    "minifyAliases": false,
    "native": false,
    "omitNull": false,
    "pool": {},
    "port": 5432,
    "protocol": "tcp",
    "query": {},
    "quoteIdentifiers": true,
    "replication": false,
    "retry": {
      "match": [
        "SQLITE_BUSY: database is locked",
      ],
      "max": 5,
    },
    "ssl": undefined,
    "standardConformingStrings": true,
    "storage": "/Users/kenedy-/projects/mvpapplication/src/utils/mvpapplication.sqlite",
    "sync": {},
    "timezone": "+00:00",
    "transactionType": "DEFERRED",
    "typeValidation": false,
  },
  "queryInterface": PostgresQueryInterface {
    "queryGenerator": PostgresQueryGenerator {
      "_dialect": PostgresDialect {
        "connectionManager": ConnectionManager {
          "config": {
            "database": "mvpapplication",
            "dialectModule": null,
            "dialectModulePath": null,
            "dialectOptions": undefined,
            "host": "127.0.0.1",
            "keepDefaultTimezone": undefined,
            "native": false,
            "password": "password",
            "pool": {
              "acquire": 60000,
              "evict": 1000,
              "idle": 10000,
              "max": 5,
              "min": 0,
              "validate": [Function],
            },
            "port": 5432,
            "protocol": "tcp",
            "replication": false,
            "ssl": undefined,
            "username": "root",
          },
          "dialect": [Circular],
          "dialectName": "postgres",
          "enumOids": {
            "arrayOids": [],
            "oids": [],
          },
          "lib": PG {
            "Client": [Function],
            "Connection": [Function],
            "DatabaseError": [Function],
            "Pool": [Function],
            "Query": [Function],
            "_pools": [],
            "defaults": {
              "application_name": undefined,
              "binary": false,
              "client_encoding": "",
              "connect_timeout": 0,
              "connectionString": undefined,
              "database": undefined,
              "fallback_application_name": undefined,
              "host": "localhost",
              "idleTimeoutMillis": 30000,
              "idle_in_transaction_session_timeout": false,
              "keepalives": 1,
              "keepalives_idle": 0,
              "lock_timeout": false,
              "max": 10,
              "options": undefined,
              "parseInputDatesAsUTC": false,
              "parseInt8": undefined,
              "password": null,
              "port": 5432,
              "query_timeout": false,
              "rows": 0,
              "ssl": false,
              "statement_timeout": false,
              "user": "kenedy-",
            },
            "escapeIdentifier": [Function],
            "escapeLiteral": [Function],
            "types": {
              "arrayParser": {
                "create": [Function],
              },
              "builtins": {
                "ABSTIME": 702,
                "ACLITEM": 1033,
                "BIT": 1560,
                "BOOL": 16,
                "BPCHAR": 1042,
                "BYTEA": 17,
                "CHAR": 18,
                "CID": 29,
                "CIDR": 650,
                "CIRCLE": 718,
                "DATE": 1082,
                "FLOAT4": 700,
                "FLOAT8": 701,
                "GTSVECTOR": 3642,
                "INET": 869,
                "INT2": 21,
                "INT4": 23,
                "INT8": 20,
                "INTERVAL": 1186,
                "JSON": 114,
                "JSONB": 3802,
                "MACADDR": 829,
                "MACADDR8": 774,
                "MONEY": 790,
                "NUMERIC": 1700,
                "OID": 26,
                "PATH": 602,
                "PG_DEPENDENCIES": 3402,
                "PG_LSN": 3220,
                "PG_NDISTINCT": 3361,
                "PG_NODE_TREE": 194,
                "POLYGON": 604,
                "REFCURSOR": 1790,
                "REGCLASS": 2205,
                "REGCONFIG": 3734,
                "REGDICTIONARY": 3769,
                "REGNAMESPACE": 4089,
                "REGOPER": 2203,
                "REGOPERATOR": 2204,
                "REGPROC": 24,
                "REGPROCEDURE": 2202,
                "REGROLE": 4096,
                "REGTYPE": 2206,
                "RELTIME": 703,
                "SMGR": 210,
                "TEXT": 25,
                "TID": 27,
                "TIME": 1083,
                "TIMESTAMP": 1114,
                "TIMESTAMPTZ": 1184,
                "TIMETZ": 1266,
                "TINTERVAL": 704,
                "TSQUERY": 3615,
                "TSVECTOR": 3614,
                "TXID_SNAPSHOT": 2970,
                "UUID": 2950,
                "VARBIT": 1562,
                "VARCHAR": 1043,
                "XID": 28,
                "XML": 142,
              },
              "getTypeParser": [Function],
              "setTypeParser": [Function],
            },
          },
          "nameOidMap": {},
          "oidParserMap": Map {},
          "pool": Pool {
            "_availableObjects": [],
            "_count": 0,
            "_draining": false,
            "_factory": {
              "acquireTimeoutMillis": 60000,
              "create": [Function],
              "destroy": [Function],
              "idleTimeoutMillis": 10000,
              "max": 5,
              "maxUses": undefined,
              "min": 0,
              "name": "sequelize",
              "reapIntervalMillis": 1000,
              "validate": [Function],
            },
            "_inUseObjects": [],
            "_pendingAcquires": [],
            "_removeIdleScheduled": false,
            "acquireTimeoutMillis": 60000,
            "idleTimeoutMillis": 10000,
            "log": false,
            "maxUsesPerResource": Infinity,
            "reapIntervalMillis": 1000,
          },
          "sequelize": [Circular],
          "versionPromise": null,
        },
        "queryGenerator": [Circular],
        "queryInterface": [Circular],
        "sequelize": [Circular],
      },
      "_quoteIdentifier": [Function],
      "dialect": "postgres",
      "options": {
        "attributeBehavior": "throw",
        "benchmark": false,
        "database": "mvpapplication",
        "databaseVersion": 0,
        "define": {},
        "dialect": "postgres",
        "dialectModule": null,
        "dialectModulePath": null,
        "hooks": {},
        "host": "127.0.0.1",
        "isolationLevel": null,
        "logQueryParameters": false,
        "logging": false,
        "minifyAliases": false,
        "native": false,
        "omitNull": false,
        "pool": {},
        "port": 5432,
        "protocol": "tcp",
        "query": {},
        "quoteIdentifiers": true,
        "replication": false,
        "retry": {
          "match": [
            "SQLITE_BUSY: database is locked",
          ],
          "max": 5,
        },
        "ssl": undefined,
        "standardConformingStrings": true,
        "storage": "/Users/kenedy-/projects/mvpapplication/src/utils/mvpapplication.sqlite",
        "sync": {},
        "timezone": "+00:00",
        "transactionType": "DEFERRED",
        "typeValidation": false,
      },
      "quoteIdentifier": [Function],
      "sequelize": [Circular],
      "typeValidation": undefined,
    },
    "sequelize": [Circular],
  },
}
`;
